### PR TITLE
Add John Kjell to TLs

### PR DESCRIPTION
**Pull request by anvega**
adds @jkjell  
adds three tables to the README to better organize the list of Security TAG members

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,34 +122,37 @@ seen [here](governance/related-groups/README.md)
...
-- Ragashree Shekar ([@ragashreeshekar](https://github.com/ragashreeshekar)), Independent
-- Michael Lieberman ([@mlieberman85](https://github.com/mlieberman85)), Kusari
+| Name                  | Organization            | Handle              |
+|-----------------------|------------------------|---------------------|
+| Justin Cappos         | New York University    | @JustinCappos       |
+| Ash Narkar            | Styra                  | @ashutosh-narkar    |
+| Andrés Vega           | M42                    | @anvega             |
+| Ragashree Shekar      | Independent            | @ragashreeshekar    |
+| Michael Lieberman     | Kusari                 | @mlieberman85       |
+| John Jkell    | TestifySec                 | @jkjell       |
```
Comment (JustinCappos) [inline README.md:141]: typo
```suggestion
| John Kjell    | TestifySec                 | @jkjell       |
```

**Review by JustinCappos — CHANGES_REQUESTED** [review]
typo

File: README.md (hunk 1 above)
Comment (anvega) [reply README.md]: Good catch!

File: README.md [hunk 2]
```diff
@@ -122,34 +122,37 @@ seen [here](governance/related-groups/README.md)
 
 ### Security TAG Chairs
 
-- Pushkar Joglekar ([@PushkarJ](https://github.com/PushkarJ)), Independent [Chair term: 6/3/2023 - 6/3/2025]
-- Marina Moore ([@mnm678](https://github.com/mnm678)), NYU [Chair term: 10/3/2023 - 10/3/2025]
-- Eddie Knight ([@eddie-knight](https://github.com/eddie-knight)), Sonatype [Chair term: 5/6/2024 - 5/6/2026]
+| Name                  | Organization            | Term                | Handle    |
+|-----------------------|------------------------|---------------------|-----------|
+| Pushkar Joglekar      | Independent            | June, 2023 - June, 2025 | @PushkarJ |
+| Marina Moore          | NYU                    | October, 2023 - October, 2025 | @mnm678   |
```
Comment (mnm678) [inline README.md:128]:
```suggestion
| Marina Moore          | Independent                  | October, 2023 - October, 2025 | @mnm678   |
```

**Review by mnm678 — COMMENTED** [review]
Other than the linting error, lgtm

**Review by PushkarJ — APPROVED** [review]
Preview looks good on website from mobile too

**Review by JustinCappos — APPROVED** [review]

**Pull request merged**